src/frontends/tensorflow/src/meta.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <stdlib.h>
+
+#include <fstream>
+#include <string>
+
+#include "graph_iterator_meta.hpp"
+#include "openvino/core/type/element_type.hpp"
+#include "tensor_bundle.pb.h"
+#include "trackable_object_graph.pb.h"
+
+#ifdef ENABLE_SNAPPY_COMPRESSION
+#    include "snappy.h"
```

Comment:
How is snappy used in this file? Is this include really needed?

Reply:
Good catch. It is artefact when code was in saved_model.cpp. Now it is used only in variables_index.cpp. I'll remove it in both files. I think it'll be useless in future too.